explainer.md

```diff
@@ -275,29 +313,59 @@ Acquire performs the following steps:
 
 Note that the acquire call takes options, consisting of the following key-value
 pairs:
-* (optional) timeout: timeoutInMilliseconds
+* (optional) `timeout`: `timeoutInMilliseconds`
   * This is an optional timeout value for the duration of the lock. When the
-    lock is acquired, and commit is not called within timeoutInMilliseconds
+    lock is acquired, and commit is not called within `timeoutInMilliseconds`
     milliseconds, the lock will be automatically committed. This is a safety
     valve for protecting pages from inadvertently leaving an element locked
     indefinitely due to uncaught exceptions or other bugs in the code. For a
     more sophisticated use cases, the value of Infinity will effectively disable
     the timeout.
+* (optional) `activatable`: `true|false`
+  * Signifies whether the user-agent can automatically unlock/commit the lock
+    when the browser needs to *activate* the locked element or its descendants.
+    See [element activation section](#element-activation) for more details.
+  * Not setting this, or setting this to `false`,
+    means this element and its descendants will be ignored in cases like
+    find-in-page, anchor link navigation, focus navigation, etc.
+
+#### DisplayLockContext.locked
 
-##### DOM modifications in the locked state
+This is a boolean value that will be `true`
+if the `Element` associated with this `DisplayLockContext` is in the locked state,
+and `false` if not.
 
-When the element is in the locked state, it can be inserted into the DOM. The
-script is then free to make changes to the element's subtree. Specifically
-changes to DOM, or style that affect its subtree update the DOM in such a way
-that script can inspect it immediately. It is important to note that style and
-layout inducing properties (see
-[what-forces-layout](https://gist.github.com/paulirish/5d52fb081b3570c81e3a)),
-when queried, will force style or layout synchronously in order to return correct
-values. When the element is painted, the current state of the DOM is not visible
-to the user. In lock-before-append mode, this means that the element is
-effectively display: none.
+#### The locked state
 
-##### DisplayLockContext.update()
+#### What happens
+
+The element and its subtree's paint output (i.e. the output of the
+layout and paint phase) will not change while the lock is acquired. In the
+lock-before-append case, this means that the visual state is empty. What this
+allows is for the user-agent to not have to finish processing the locked
+element’s subtree when processing the layout and paint phase.
+
+In other words, when requested, the user-agent is free to process parts of the subtree,
```

Comment:
This paragraph is I think hard to understand for non-experts. Is it really needed?